resources/js/Pages/Rating/Rating.tsx

```diff
@@ -0,0 +1,42 @@
+import {Inertia} from '@inertiajs/inertia'
+import {ThemeProvider} from '@mui/material/styles'
+import {RatingTable} from 'Components'
+import {ApplicationLayout} from 'Layouts'
+import React, {useState} from 'react'
+import {ReportedRatingData} from 'Types/reported-rating-data'
+import {getFontTheme} from 'Utils'
+import route from 'ziggy-js'
+
+const theme = getFontTheme('Form', 16)
+
+type RatingPropType = {
+    users: ReportedRatingData[]
+}
+
+const Rating = ({users}: RatingPropType) => {
+    const [isRatingFilteredForEvent, setIsRatingFilteredForEvent] =
+        useState<boolean>(false)
+    const filterTable = () => {
+        const newFilterState = !isRatingFilteredForEvent
+        setIsRatingFilteredForEvent(newFilterState)
```

Comment:
don't really understand the 2 lines here

Reply:
For some reason the state hook is not being called before moving on to the next command which needs the boolean in its right state..